src/node_zlib.cc

```diff
@@ -43,7 +43,6 @@ enum node_zlib_mode {
 
 #define GZIP_HEADER_ID1 0x1f
 #define GZIP_HEADER_ID2 0x8b
```

Comment:
Yep, but #5884 would start using them again, so I’d leave them in unless it gets decided that that should not get landed.